components/alert/alert.component.ts

```diff
@@ -24,15 +25,19 @@ import { takeUntil } from 'rxjs/operators';
 
 import { slideAlertMotion } from 'ng-zorro-antd/core/animation';
 import { NzConfigKey, NzConfigService, WithConfig } from 'ng-zorro-antd/core/config';
+import { NzOutletModule } from 'ng-zorro-antd/core/outlet';
 import { BooleanInput } from 'ng-zorro-antd/core/types';
 import { InputBoolean } from 'ng-zorro-antd/core/util';
+import { NzIconModule } from 'ng-zorro-antd/icon';
 
 const NZ_CONFIG_MODULE_NAME: NzConfigKey = 'alert';
 
 @Component({
   selector: 'nz-alert',
   exportAs: 'nzAlert',
   animations: [slideAlertMotion],
+  standalone: true,
+  imports: [BidiModule, NgIf, NzIconModule, NzOutletModule],
```

Comment:
suggestion: You can remove BidModule because the directive exported by this module is not used here